specification/communication/data-plane/CallingServer/preview/2022-04-07-preview/communicationservicescallingserver.json

```diff
@@ -0,0 +1,1242 @@
+          },
+          {
+            "name": "addParticipantRequest",
+            "in": "body",
+            "description": "The add participant request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/AddParticipantRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Returns the add participant response.",
+            "schema": {
+              "$ref": "#/definitions/AddParticipantResponse"
+            }
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "401": {
+            "$ref": "#/responses/401"
+          },
+          "403": {
+            "$ref": "#/responses/403"
+          },
+          "404": {
+            "$ref": "#/responses/404"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        },
+        "x-ms-examples": {
+          "addParticipant": {
+            "$ref": "./examples/CallConnection_AddParticipant.json"
+          }
+        }
+      }
+    },
+    "/calling/callConnections/{callConnectionId}/participants:remove": {
+      "post": {
+        "tags": [
+          "ParticipantsManagement"
+        ],
+        "summary": "Remove participants from a call.",
+        "description": "Remove participants from a call.",
+        "operationId": "CallConnection_RemoveParticipant",
+        "parameters": [
+          {
+            "name": "callConnectionId",
+            "in": "path",
+            "description": "The call connection ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "removeParticipantRequest",
+            "in": "body",
+            "description": "The remove participant request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RemoveParticipantRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Returns the remove participant response.",
+            "schema": {
+              "$ref": "#/definitions/RemoveParticipantResponse"
+            }
+          },
+          "400": {
+            "$ref": "#/responses/400"
+          },
+          "401": {
+            "$ref": "#/responses/401"
+          },
+          "403": {
+            "$ref": "#/responses/403"
+          },
+          "404": {
+            "$ref": "#/responses/404"
+          },
+          "500": {
+            "$ref": "#/responses/500"
+          }
+        },
+        "x-ms-examples": {
+          "removeParticipant": {
+            "$ref": "./examples/CallConnection_RemoveParticipant.json"
+          }
+        }
+      }
+    },
+    "/calling/callConnections/{callConnectionId}/participants:get": {
```

Comment:
To the point Jeff makes, does this make sense to model the participants as a collection? In which case, the get on the collection would be the same as Get above and there would be another get with the ID of the participant?

Reply:
This endpoint is for getting a single participant, and as Jeff also commented this is a POST because we require caller to specify the participant in the body. This is because the ID can be in different forms, it can be an MRI or a phone number.

The GET endpoint is for getting a collection of participants.

Reply:
This has now been address as GET endpoint and not a POST with :get verb